RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -343,6 +343,22 @@ private struct RoundedCorner: Shape {
 
 #endif
 
+// MARK: - Disabling Refreshable
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+extension View {
+
+    @ViewBuilder
+    func refreshableDisabled() -> some View {
+        if let refreshKeyPath = \EnvironmentValues.refresh as? WritableKeyPath<EnvironmentValues, RefreshAction?> {
+            self.environment(refreshKeyPath, nil)
+        } else {
+            self
+        }
+    }
+
+}
```

Comment:
I think this only works if the refreshable is applied before this, right?

Reply:
Right. But that's the use case we need. To disable any `refreshable` that the `PaywallView` might inherit from the caller